osx/release-docs/RELEASE_NOTES.md

```diff
@@ -1,6 +1,28 @@
 Release Notes for KA Lite Mac OS X Application
 ==============================================
 
+0.17.0
+------
+
+**Mac Installer**
+
+* NEW - We now support Mac OS Sierra.
+* NEW - We now required Python 2.7.11+ installed on your machine to install KA Lite.
```

Comment:
Can you rephrase this please?  Try "We now require Python 2.7.12 or higher to install KA Lite."?